packages/mui-joy/src/Input/Input.tsx

```diff
@@ -124,6 +124,7 @@ const InputInput = styled('input', {
   alignSelf: 'stretch',
   color: 'inherit',
   backgroundColor: 'transparent',
+  fontFamily: 'inherit',
```

Comment:
Make sure that the input font changes if the `fontFamily.body` is customized in the theme.